### PR TITLE
Update ItemDisplay to use em instead of px

### PR DESCRIPTION
**Pull request by BenBoersma**
### Summary
Updating ItemDisplay to better scale by using em for minimums. 
Resolves https://github.com/cerner/terra-clinical/issues/244 and Resolves https://github.com/cerner/terra-clinical/issues/259

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

**Comment by neilpfeiffer** [comment]
Displays as intended

Marking as Verified, ready for Release.
 
**Pass** |
-----------|
UX Review|

File: packages/terra-clinical-item-display/src/ItemDisplay.scss [hunk 1]
```diff
@@ -17,12 +17,12 @@
   .inline-icon {
     flex: 0 0 auto;
     overflow: hidden;
-    padding-right: 5px;
+    padding-right: 0.357142857142857rem;
```
Comment (yuderekyu) [inline packages/terra-clinical-item-display/src/ItemDisplay.scss]: so precise haha

**Comment by emilyrohrbough** [comment]
Did you remove all screenshots generated under `__screenhots__/reference/chrome/`?

**Comment by BenBoersma** [comment]
Yes. I was told that was how to regenerate the new screenshots references.

**Comment by emilyrohrbough** [comment]
With the toolkit changes, the entire filepath of the screenshots updates, so as long as nothing is under `__screenhots__/reference/chrome/` then the merge was successful

**Pull request merged**